TACC/tup-ui
# feat(tup-cms): cmd news styles

## Overview

Update for CMD News styles... which, for code in this repository, is just a settings change to hide tags.

<details>

The changes have been mostly in https://github.com/TACC/Core-CMS and in Snippets on https://dev.tup.tacc.utexas.edu .

</details>

## Related

- [TUP-395](https://jira.tacc.utexas.edu/browse/TUP-395)
- requires https://github.com/TACC/Core-CMS/pull/587
- requires several commits to https://github.com/TACC/Core-CMS `main` branch

## Changes

- changed styles via https://github.com/TACC/Core-CMS/pull/587
- changed `TACC_BLOG_SHOW_TAGS` setting to `False`.

## Testing

1. Add tag or tags to an article.
2. View article in list.
3. Verify tag is not shown on article.
4. View article page.
5. Verify tag is not shown on article.

See https://github.com/TACC/Core-CMS/pull/587.

## UI

<details><summary>Unorganized Screenshots</summary>

![Screen Shot 2023-02-09 at 14 50 52](https://user-images.githubusercontent.com/62723358/217936659-8fe1478c-9e80-4251-b2be-d7efa61bff22.png)
![Screen Shot 2023-02-09 at 14 51 17](https://user-images.githubusercontent.com/62723358/217936665-bc4dd64f-0dde-4820-8c21-70c076ac4714.png)
![Screen Shot 2023-02-09 at 14 51 40](https://user-images.githubusercontent.com/62723358/217936666-962e3169-2618-4c38-bd90-7496695da573.png)
![Screen Shot 2023-02-09 at 14 51 45](https://user-images.githubusercontent.com/62723358/217936667-9915a8c5-2544-40e7-91e9-aedb974dc7be.png)
![Screen Shot 2023-02-09 at 14 51 48](https://user-images.githubusercontent.com/62723358/217936669-a0b8c6a4-0e62-4fc5-872e-7977bf058383.png)
![Screen Shot 2023-02-09 at 14 51 52](https://user-images.githubusercontent.com/62723358/217936670-b80df887-4b96-47ed-946b-acbb034218d2.png)
![Screen Shot 2023-02-09 at 14 52 12](https://user-images.githubusercontent.com/62723358/217936671-e9b296fb-31f6-4dd1-a91b-d5a7e3222e97.png)
![Screen Shot 2023-02-09 at 14 52 19](https://user-images.githubusercontent.com/62723358/217936672-75566f25-14fc-4450-b9ed-bb2a3497ff1a.png)
![Screen Shot 2023-02-09 at 14 52 22](https://user-images.githubusercontent.com/62723358/217936673-ba990273-db3d-44c2-afdf-07ab9afc80c3.png)

</details>
